documentation/source/user_section/tutorials/shape-analysis/cross-sectional-area.rst

```diff
@@ -0,0 +1,17 @@
+.. _cross-sectional-area:
+
+Cross-sectional area (CSA)
+######################################################
+
+This section demonstrates how to compute spinal cord cross-sectional area.
+
+
+.. toctree::
+   :maxdepth: 1
+
+   cross-sectional-area/csa-averaged
+   cross-sectional-area/csa-perlevel
+   cross-sectional-area/csa-pmj
+   cross-sectional-area/csa-perslice
+
```

Comment:
my suggestion would be to keep all four cases inside the same page-- 
pros: 
- each one is very short, so it's no problem to aggregate them into one page
- easier from code management standpoint
- easier for user (less clicks)
- clearer for user (less jammed TOC)